canonical-web-and-design/react-components
# Prepare release 0.32.0

## Done

- Generated docs for new version
- Bumped version in package.json

## QA

- Make sure version is correct
- Storybook files are generated, no need to code review those
- Make sure demo is running fine https://react-components-721.demos.haus/
